nsc2_releasenotes.md

```diff
@@ -12,28 +12,28 @@ You can easily move between releases by doing `nsc update --version XXX`, where
 
 The upgrade procedure requires the following steps:
 
-- Upgrade your nats-servers to 2.2.0 or better. The nats-server will work correctly with your previous JWT configurations.
+- Upgrade your nats-server(s) to 2.2.0 or better. The nats-server will work correctly with your previous JWT configurations.
 
-- Do verify that all your servers are on the same version, if using a [`nats-account-server`](https://docs.nats.io/nats-tools/nas) make sure the version you upgrade to is JWT 2 compliant.
+- Verify all of your servers are on the same version, if using a [`nats-account-server`](https://docs.nats.io/nats-tools/nas) make sure the version you upgrade to is JWT 2 compliant.
 
-- Upgrade your nsc configurations. Use the `nsc upgrade-jwt` command. You will need the Operator's main identity key to perform the upgrade.
+- Upgrade your NSC configurations. Use the `nsc upgrade-jwt` command. You will need the Operator's main identity key to perform the upgrade.
 
-- Redistribute the updated Operator JWT, to all your servers. You can easily export the operator JWT by issuing the command. If your server is configured to a JWT operator file, make sure the new file is referenced. If the JWT is embedded into the server configuration, make sure to copy the contents of an exported operator JWT. To example the operator to a file, you can execute the command: `nsc describe operator --raw > /tmp/operator.jwt`.
+- Redistribute the updated Operator JWT, to all of your servers. You can easily export the operator JWT by issuing the command. If your server is configured to a JWT operator file, make sure the new file is referenced. If the JWT is embedded into the server configuration, make sure to copy the contents of an exported operator JWT. To example the operator to a file, you can execute the command: `nsc describe operator --raw > /tmp/operator.jwt`.
```

Comment:
@aricart The sentence 'To example the operator to a file...'  Is 'example' the correct word here?

Reply:
sigh - no - "For example, to save your operator to a file..."